docs/UserGuide.md

```diff
@@ -611,14 +627,12 @@ Important notes:
 > Location: Clarke Quay;
 > Categories: #social
 > ```
-> Input:
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteTransactionInitialState.png" width="1000" />
-> Output:
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteTransactionSuccess1FinalState.png" width="1000" />
 
 ###### Example 2
 
-> **Case**: Delete a transaction with the currently not visible but correctly specified `<INDEX>`.
+> **Case**: Delete a transaction with the currently not visible but correctly specified
+> `INDEX`. The transaction is present in the current `Transactions List` configuration
+> but the user has scrolled further away. 
```

Comment:
I don't quite get the purpose of this example and its relevant screenshots, seems like this could be kept as a callout above indicating that if delete can be triggered even if a transaction is out of the visible window.

Reply:
Makes sense, I've removed both